lib/collection.js

```diff
@@ -2477,6 +2477,7 @@ Collection.prototype.aggregate = function(pipeline, options, callback) {
  * @param {number} [options.batchSize=null] The number of documents to return per batch. See {@link https://docs.mongodb.com/manual/reference/command/aggregate|aggregation documentation}.
  * @param {object} [options.collation=null] Specify collation settings for operation. See {@link https://docs.mongodb.com/manual/reference/command/aggregate|aggregation documentation}.
  * @param {ReadPreference} [options.readPreference=null] The read preference. Defaults to the read preference of the database or collection. See {@link https://docs.mongodb.com/manual/reference/read-preference|read preference documentation}.
+ * @param {Timestamp} [options.startAtClusterTime] receive change events that occur after the specified timestamp
```

Comment:
this is a BSON Timestamp presumably, should we link accordingly?

Reply:
I believe this will link to BSON:Timestamp, though I will have to test it out.

Reply:
Confirmed that it links to the Timestamp page in the docs